src/main/scala/parsec/CharParsers.scala

```diff
@@ -29,10 +29,9 @@ object HelloCharParsers extends CharParsers {
 
   val (strZ, strCombine) = stringFolder
 
-  val wordDigitLetter: Parser[(String, Char)]
-    = (letters ~ digit).fold(strZ, strCombine) map {
-      case (ls, other) => (ls.toString, other)
-    }
+  val wordDigitLetter: Parser[(String, Char)] = (letters ~ digit).fold(strZ, strCombine) map {
+    case (ls, other) => (ls.toString, other)
+  }
 
```

Comment:
gotta say I prefer the left hand side indentation, the right hand side crosses 80 characters. But it could be styled better, I agree. According to sjrd's [guide](https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md) we'd get:

```scala
val wordDigitLetter: Parser[(String, (Char, Char))] = {
  (letters ~ digit ~ letter).fold(strZ, strCombine) map {
    case (ls, other) => (ls.toString, other)
  }
}
```